src/openzaak/import_data/models.py

```diff
@@ -0,0 +1,140 @@
+# SPDX-License-Identifier: EUPL-1.2
+# Copyright (C) 2019 - 2024 Dimpact
+import uuid
+
+from django.conf import settings
+from django.db import models
+from django.urls import reverse
+from django.utils.functional import classproperty
+from django.utils.translation import gettext as _
+
+from privates.fields import PrivateMediaFileField
+from vng_api_common.constants import ComponentTypes
+
+from openzaak.utils import build_absolute_url
+
+
+class ImportStatusChoices(models.TextChoices):
+    pending = "pending", _("Openstaand")
+    active = "active", _("Actief")
+    finished = "finished", _("Voltooid")
+    error = "error", _("Onderbroken")
+
+    @classproperty
+    def visible_choices(cls):
+        return {cls.pending, cls.active, cls.finished, cls.error}
+
+    @classproperty
+    def started_choices(cls):
+        return {cls.pending, cls.active}
+
+    @classproperty
+    def report_choices(cls):
+        return {cls.finished, cls.error}
+
+
+class ImportRowResultChoices(models.TextChoices):
+    imported = "imported", _("Geïmporteerd")
+    not_imported = "not_imported", _("Niet geïmporteerd")
+
+
+class ImportTypeChoices(models.TextChoices):
+    documents = "documenten", _("Enkelvoudige informatie objecten")
+
+    @classproperty
+    def component_mapping(cls):
+        return {
+            cls.documents: ComponentTypes.drc,
+        }
+
+    @classmethod
+    def get_component_from_choice(cls, choice):
+        return cls.component_mapping[choice]
+
+
+class Import(models.Model):
```

Comment:
Could you please add this model into `fixtures/default_admin_index.json` ? So it will appear in the admin